### PR TITLE
Fax responder renovation

### PR DESCRIPTION
**Pull request by Syndro101**
# About the pull request

Completely renovates the offices used by fax responders to a much higher quality.

# Explain why it's good for the game

Adds more depth to the job other than being stuck in a cubicle and looks nice.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

StrongDMM-2024-12-07 11 02 41

</details>

# Changelog

:cl:
add: Added a new area for the sector relay.
mapadd: Added new offices for Fax Responders.
maptweak: Renamed the old map.
fix: Fax responders no longer count to round end.
/:cl:

### CODE REVIEW

**Comment by thevandie** [comment]
you should give the WY responder a Liaison's Wardrobe so he can drip himself out

File: code/game/area/admin_level.dm
Comment (Nanu308) [inline code/game/area/admin_level.dm:1]: lgtm

File: code/game/turfs/open.dm
Comment (Nanu308) [inline code/game/turfs/open.dm:1]: lgtm

File: code/game/turfs/shiva.dm
Comment (Nanu308) [inline code/game/turfs/shiva.dm:1]: lgtm

File: maps/templates/lazy_templates/fax_responder_base.dmm
Comment (Nanu308) [inline maps/templates/lazy_templates/fax_responder_base.dmm:1]: image
Table can be climbed across? Unsure if intended or not, consider just leaving a door between HC and Provost.

**Review by Nanu308 — CHANGES_REQUESTED** [review]
Very good changes and environmentally fitting areas made for each faction here. Well done overall 👍 

See comment on HC/Provost, other than that mapping lgtm

File: maps/templates/lazy_templates/fax_responder_base.dmm
Comment (Syndro101) [reply maps/templates/lazy_templates/fax_responder_base.dmm:1]: Double-checked, no issue and the table remains unclimbable.

**Comment by Syndro101** [comment]
Everything should be ready to go.

**Review by Nanu308 — APPROVED** [review]
mapping lgtm

File: code/game/bioscans.dm [hunk 1]
```diff
@@ -70,6 +72,8 @@ GLOBAL_DATUM_INIT(bioscan_data, /datum/bioscan_data, new)
 	for(var/mob/living/carbon/human/current_human as anything in GLOB.alive_human_list)
 		if(current_human.mob_flags & NOBIOSCAN)
 			continue
+		if(istype(get_turf(current_human), /area/adminlevel/ert_station/fax_response_station))
+			continue
```
Comment (harryob) [inline code/game/bioscans.dm:76]: these should be flags or traits on the area that make them completely immune to bioscans, not snowflake type checks

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/game/bioscans.dm (hunk 1 above)
Comment (Syndro101) [reply code/game/bioscans.dm]: imported forest code

**Pull request merged**